crypto/bio/bss_dgram.c

```diff
@@ -573,6 +575,10 @@ static long dgram_ctrl(BIO *b, int cmd, long num, void *ptr)
         b->shutdown = (int)num;
         b->init = 1;
         dgram_update_local_addr(b);
+	if (getpeername(b->num, (struct sockaddr *)&ss, &ss_len) == 0) {
```

Comment:
indentation needs fixing in this file

Reply:
Ah, @Sashan it seems your editor needs to be configured to replace tabs with spaces.